Add CSS revert-rule keyword (#43808)
mdn/content: files/en-us/web/css/reference/values/revert/index.md

- Use the {{cssxref("initial")}} keyword to set a property to its initial value.
- Use the {{cssxref("inherit")}} keyword to make an element's property the same as its parent.
- Use the {{cssxref("revert-layer")}} keyword to reset a property to the value established in a previous cascade layer.

> - Use the {{cssxref("revert-rule")}} keyword to reset a property as if the current style rule were not present.

Reviewer: ```suggestion
- Use the {{cssxref("revert-rule")}} keyword to reset a property to the value of a previous matching style rule.
```
